bc_obps/registration/schema/operation.py

```diff
@@ -20,9 +20,9 @@ class OperationCreateIn(ModelSchema):
     # operation_has_multiple_operators: Optional[bool] = False
     # multiple_operators_array: Optional[list] = None
 
-    class Meta:
+    class Config:
```

Comment:
@Sepehr-Sobhani , this change caused four of the backend tests to fail because naics code was coming out as None. Thoughts?